native/core/src/parquet/schema_adapter.rs

```diff
@@ -216,57 +216,6 @@ impl SchemaMapper for SchemaMapping {
         let record_batch = RecordBatch::try_new_with_options(schema, cols, &options)?;
         Ok(record_batch)
     }
-
-    /// Adapts a [`RecordBatch`]'s schema into one that has all the correct output types and only
-    /// contains the fields that exist in both the file schema and table schema.
-    ///
-    /// Unlike `map_batch` this method also preserves the columns that
-    /// may not appear in the final output (`projected_table_schema`) but may
-    /// appear in push down predicates
-    fn map_partial_batch(&self, batch: RecordBatch) -> datafusion::common::Result<RecordBatch> {
```

Comment:
`map_partial_batch` has been removed in DataFusion

Reply:
@mbutrovich @parthchandra fyi, I am not sure of the impact, but I figured I would first see if any tests fail in CI